spikeextractors/extractors/nwbextractors/nwbextractors.py

```diff
@@ -6,13 +6,13 @@
 import distutils.version
 from typing import Union, List, Optional
 import warnings
+import pandas as pd
```

Comment:
Can you keep pandas in the `try`? It's not installed by default )https://github.com/SpikeInterface/spikeextractors/blob/master/setup.py#L20-L24)